ConwaysGameOfLife.Tests/UniverseHelper.cs

```diff
@@ -84,15 +84,15 @@ public static bool LocationsAreEqual(Location location1, Location location2)
                 for (var x = 0; x < gridWidth; x++)
                 {
                     var currentCell = row[x];
-                    cells.Add(new Cell(GetSourceDataCellState(currentCell), x, y));
+                    cells.Add(new Cell(GetCellState(currentCell), x, y));
                 }
             }
             return new Universe(gridWidth, gridLength, cells);
         }
 
-        private static CellState GetSourceDataCellState(char value) // TODO: rename method and parameter
+        private static CellState GetCellState(char input) // TODO: rename method and parameter 9/6
         {
-            if (value == 'X')
+            if (input == 'X')

```

Comment:
```suggestion
            return input == 'O' ? CellState.Alive : CellState.Dead;
```

think my comment on a previous PR may suggest otherwise, but not sure if we want to consider if `input` isn't `X` or `O`.